filters.go

```diff
@@ -311,6 +311,38 @@ func (p *Pipe) MatchRegexp(re *regexp.Regexp) *Pipe {
 	})
 }
 
+// ReadInput reads displays the pipe content as a message as a prompt
+// and reads the content from standard input and returns the result
+// as a pipe. In case the input was empty it defaults to its `defaultValue`
+func (p *Pipe) ReadInput(defaultValue string) *Pipe {
+	const emptyString = ""
```

Comment:
There's really no value in defining a constant for empty string, is there? We can just use an empty literal instead.

Reply:
Of course, I was just thinking that reading the code would flow a bit better.